resource/abstractions/auditing_rules.md

```diff
@@ -0,0 +1,478 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Abstractions
+---
+
+# Auditing Rules
+
+General Resource Profile Administration functions for manipulating a General Resource Profile's Auditing Rules. 
+{: .fs-6 .fw-300 }
+
+## `ResourceAdmin.get_audit_rules()`
+
+```python
+def get_audit_rules(self, resource: str, class_name: str) -> Union[dict, bytes, None]:
+```
+
+#### 📄 Description
+
+Check the **Audit Rules** for a general resource profile.
+
+#### 📥 Parameters
+* `resource`<br>
+  The **general resource profile** for which RACF should check the audit rules.
+
+* `class_name`<br>
+  The name of the **class** the general resource profile being checked belongs to.
+
+#### 📤 Returns
+* `Union[dict, bytes, None]`<br>
+  Returns `None` when the general resource profile has no **Auditing Rules** defined, otherwise returns the auditing rules as a dictionary. If the `ResourceAdmin.generate_requests_only` class attribute is set to `True`, **concatenated Security Request XML bytes** will be returned.
+
+#### ❌ Raises
+* `SecurityRequestError`<br>
+  Raises `SecurityRequestError` when the **SAF Return Code** of a **Security Result** is equal to `4`.
+* `DownstreamFatalError`<br>
+  Raises `DownstreamFatalError` when the **SAF Return Code** of a **Security Result** is greater than `4`.
+
+#### 💻 Example
+
+###### Python REPL
+```python
+>>> from pyracf import ResourceAdmin
+>>> resource_admin = ResourceAdmin()
+>>> resource_admin.get_audit_rules("TESTING","ELIJTEST")
+{"success": "update", "failures": "read"}
+```
+
+## `ResourceAdmin.clear_all_audit_rules()`
```

Comment:
I believe we renamed this to `ResourceAdmin.remove_all_audit_rules()`

Reply:
We did, apologies, that was one change that did not make it to this documentation yet. Fixed now.